uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/entitlement-instance_test.js

```diff
@@ -0,0 +1,44 @@
+define([
+  'leave-absences/shared/apis/entitlement-api',
+  'leave-absences/shared/models/entitlement-model',
+  'leave-absences/shared/models/instances/entitlement-instance',
+], function () {
+  'use strict'
+
+  describe('EntitlementInstance', function () {
+    var Entitlement, EntitlementInstance, $rootScope, $httpBackend, EntitlementAPIMock;
+
+    beforeEach(module('leave-absences.models'));
+
+    beforeEach(inject(function (_Entitlement_, _EntitlementInstance_) {
+      Entitlement = _Entitlement_;
+      EntitlementInstance = _EntitlementInstance_;
+    }));
+
+    describe('defaultCustomData()', function () {
+      it('has default values', function () {
+        var expectedDefault = {
+          remainder: {
+            current: 0,
+            future: 0
+          },
+          breakdown: []
+        };
+        expect(EntitlementInstance.defaultCustomData()).toEqual(expectedDefault);
+      });
+    });
+
+    describe('breakdown()', function () {
+      it('will get breakdown details for given entitlement', function () {
```

Comment:
done not needed added digest

'    afterEach(function () {
      //to excute the promise force an digest
      $rootScope.$apply();
    });'